src/extension/geofence/geofence/src/main/java/org/geoserver/geofence/GeofenceAccessManager.java

```diff
@@ -219,6 +221,50 @@ private boolean isWorkspaceAdmin(Authentication user, String workspaceName) {
         return auth.getAdminRights();
     }
 
+    /**
+     * Parse a String holding an IP address and optionally a port.
+     *
+     * <p>Address may be either in IPv4 or IPv6 form. Recognized formats are:
+     *
+     * <ul>
+     *   <li>ipv4
+     *   <li>ipv4:port
+     *   <li>ipv6
+     *   <li>[ipv6]
+     *   <li>[ipv6]:port
+     *   <li>hostname
+     *   <li>hostname:port
+     * </ul>
+     *
+     * This method does not perform any validation check on the format; it only tries and parse any of the above
+     * formats.
+     *
+     * @param a String representing the address
+     * @return String, where [0] holds the ip or hostname and [1] the port or null
```

Comment:
I don't see the port being used anywhere in the caller code... if that's actually the case, why bother computing it?